lib/simulator-xcode-6.js

```diff
@@ -100,16 +100,10 @@ class SimulatorXcode6 extends EventEmitter {
    * or, for iOS 7.1, the app name without `.app` (e.g., MobileSafari)
    */
   async getAppDir (id, subDir = 'Data') {
-    if (await this.isFresh()) {
-      log.info('Attempted to get an app path from a fresh simulator ' +
-               'quickly launching the sim to populate its directories');
-      await this.launchAndQuit();
```

Comment:
excellent point. if we intend to launch and quit for a particular purpose, we should do it in the place where that purpose is defined in code, not as a hidden side effect here. 👍